### PR TITLE
fix(file): confirm or correct asserted DOCX, PPTX, and XLSX content types

### PR DESCRIPTION
**Pull request by scanny**
**Summary**
The `content_type` argument received by `partition()` from the API is sometimes unreliable for MS-Office 2007+ MIME-types. What we've observed is that it gets the MS-Office bit right but falls down on distinguishing PPTX from DOCX or XLSX.

Confirmation of these types is simple, fast, and reliable. Confirm all MS-Office `content_type` argument values asserted by callers of `detect_filetype()` and correct swapped values.

### CODE REVIEW

File: test_unstructured/file_utils/test_filetype.py [hunk 1]
```diff
@@ -664,13 +696,13 @@ def and_it_returns_the_empty_string_as_the_extension_when_there_are_no_file_name
...
 
-    def it_grabs_the_first_4k_bytes_of_the_file_for_use_by_magic(self):
+    def it_grabs_the_first_8k_bytes_of_the_file_for_use_by_magic(self):
         ctx = _FileTypeDetectionContext(file_path=example_doc_path("norwich-city.txt"))
 
         head = ctx.file_head
 
         assert isinstance(head, bytes)
-        assert len(head) == 4096
+        assert len(head) == 8192
```
Comment (scanny) [inline test_unstructured/file_utils/test_filetype.py:705]: This bump of `.file_head` size to `8192` has nothing specific to do with fixing the problem remedied by this PR, it was just too small to put in it's own PR so came along for the ride.

**Review by Coniferish — APPROVED** [review]
LGTM
